workflows/cloudify_system_workflows/snapshot.py

```diff
@@ -263,21 +283,64 @@ def _update_es_node(es_node):
                 }
 
 
-def _restore_elasticsearch(ctx, tempdir, es):
+def _restore_elasticsearch(ctx, tempdir, es, metadata):
     ctx.send_event('Deleting all ElasticSearch data')
     elasticsearch.helpers.bulk(
         es,
         _clean_up_db_before_restore(es, ctx.execution_id))
     es.indices.flush()
 
-    def es_data_itr():
+    cloudify_events = es.indices.exists(index='cloudify_events')
```

Comment:
This sounds more like an iterator over cloudify events or sth like that, maybe rename this variable to events_index_exists ?